<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyWidget</class>
 <widget class="QWidget" name="modifyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="idEdit">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="deleteButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="carRadioButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Car</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="truckRadioButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Truck</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="busRadioButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bus</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="modelEdit">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="fromEdit">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Basic Fee:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="basicFee">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Road Fee:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="roadFee">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Unit Fee:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="mileUnitEdit">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>yuan/km</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mile All:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="mileAllEdit">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="modifyButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buy Time:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="dateEdit">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="returnButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Back To Menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
